star/test_suite/each_test_run

```diff
@@ -253,6 +253,12 @@ if [ "$DO_ALL" == 1 ] || [ "$curr_id" == "$id" ] ; then
     else
         testhub_log "fpe_checks: true"
     fi
+    if [ -z "$MESA_TEST_SUITE_RESOLUTION_FACTOR" ] || 
+       (( $(echo "$MESA_TEST_SUITE_RESOLUTION_FACTOR == 1" | bc -l) )); then
```

Comment:
This second part of this test seems unnecessary.  Why bother?  If the env var is set to `1.0` it seems fine to go to the other branch and write that in the YAML.  It's a presumably floating point field, so it'll all be the same when it eventually hits the testhub DB.

Reply:
Sure, I can remove it. My only thought was to save potential headache if the implementer on the other side only checked that "1" was the "off" state without accounting for e.g. "1.0000" being a value that could show up there. But I agree, it's not necessary.